### ISSUE TITLE
SSHFactory shouldn't disable core dumps

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2715]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#2715|
|Type|defect|
|Created|2007-06-24 19:47:59Z|

If core dumps should be disabled, it is someone else's responsibility to disable them.

<details><summary>Searchable metadata</summary>

```
trac-id__2715 2715
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__ 
time__1182714479000000 1182714479000000
changetime__1277603369000000 1277603369000000
version__None None
owner__ 
cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_1"></a>|@jesstess commented|
|-|-|

(In [29299]) Branching to 'sshfactory-coredumps-2715'

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_3"></a>|@jesstess commented|
|-|-|

(In [29300]) SSHFactory no longer disables coredumps.

refs [#2715](https://github.com/twisted/twisted/issues/2715)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to @jesstess|
|-|-|

 * We should have a ticket for re-adding this functionality, perhaps to `twisted.conch.tap` (perhaps enabled by default, with a flag to disable it).
 * The news fragment should use the fully-qualified name of `SSHFactory`.

Aside from that, great.  Please merge when the above two points are addressed.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_6"></a>|@jesstess commented|
|-|-|

(In [29393]) Use fully-qualified name of SSHFactory in the news file.

refs [#2715](https://github.com/twisted/twisted/issues/2715)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_7"></a>|@jesstess commented|
|-|-|

Ticket [4521](http://twistedmatrix.com/trac/ticket/4521) has been opened to re-introduce this functionality.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_8"></a>|@jesstess set status to `closed`|
|-|-|

(In [29394]) Merge sshfactory-coredumps-2715

Author: jesstess
Reviewer: exarkun
Fixes: [#2715](https://github.com/twisted/twisted/issues/2715)

twisted.conch.ssh.factory.SSHFactory no longer disables coredumps.
Ticket [#4521](https://github.com/twisted/twisted/issues/4521) was opened to re-introduce this functionality in
twisted.conch.tap.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
